core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionFixture.scala

```diff
@@ -5,17 +5,11 @@ package org.broadinstitute.dsde.rawls.expressions
 trait ExpressionFixture {
   // NOTE: empty strings are handled specially by the validator, so they don't need to be handled by the parser
 
-  val parseableInputExpressions = Seq(
-    "this.gvcf",
+  val parseableInputExpressionsWithNoRoot = Seq(
```

Comment:
just splitting up of "what's parseable if you have no root entity" and "what's parseable if you do have a root entity", since the latter is bigger than the former